content/docs/cms/events.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Events
+prev: /docs/cms/apis
+next: /docs/cms/alerts
+---
+
+The **Events** feature allows decoupled parts of the CMS to notify the rest of the system about certain events which have happened.
```

Comment:
```suggestion
The **Events** feature allows decoupled parts of the CMS to notify the rest of the system that something has happened.
```

Reply:
This sentence is kinda heavy to me. I'm not sure if my suggestion makes it better :man_shrugging:

Reply:
```suggestion
The **Events** feature allows decoupled parts of the CMS to:

  1. Notify the rest of the system that something has happened, and
  2. React to things that happen elsewhere in the system.
```